src/newLaunch/lbp/stages/stage4.html

```diff
@@ -76,7 +76,7 @@
         <div class="infoInput maxTarget fundingTarget">
 
           <div class="fundingDescrp">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit
```

Comment:
I don't think we were supposed to remove this text.  Please check with Renc about this.

Reply:
@dkent600 i asked @renc-curvelabs  and said that we need to remove it like i did

Reply:
ok